### ISSUE TITLE
symbolic link to sample-2.darshan

### ISSUE THREAD

**Issue by shanedsnyder** [darshan-hpc/darshan#298]
In GitLab by @wkliao on Feb 13, 2021, 03:45

File
```
darshan-util/pydarshan/tests/input/darshanlogs/2017/3/20/glock_dbscan_read_id4478544_3-20-7800-15217972833987807816_1.darshan
```
is a symbolic link to file
```
../../../../sample-2.darshan
```
which does not exist.

**Comment by shanedsnyder** [comment]
In GitLab by @carns on Feb 23, 2021, 11:42

@jakobluettgau, we can just just delete the `darshan-util/pydarshan/tests/input/darshanlogs` subdir (and things beneath it), right?  I don't see any references to those files, just the files directly in the `darshan-util/pydarshan/tests/input/` directory.

Tagging @kevin-harms too for input.

**Comment by shanedsnyder** [comment]
In GitLab by @jakobluettgau on Feb 23, 2021, 11:53

Ah, yes sure! That probably shouldn't have made it into the repository anyway.

**Comment by shanedsnyder** [comment]
In GitLab by @carns on Feb 23, 2021, 11:57

closed via commit 649e3d35eeac81ae6273add1951aa338cf20334f

**Comment by shanedsnyder** [comment]
In GitLab by @carns on Mar 29, 2021, 19:56

mentioned in commit 649e3d35eeac81ae6273add1951aa338cf20334f
